omero/sysadmins/unix/server-installation.txt

```diff
@@ -175,8 +174,8 @@ Check you have Python (and check its version) by running:
     $ python --version
     Python 2.7.5+
 
-.. note:: OMERO does not currently support Python 3; you must use 2.6 or
-   a later 2.x release.
+.. note:: OMERO does not currently support Python 3; you must use 2.6 or a
+    later 2.x release.
```

Comment:
I'd be tempted to get rid of the `x` and say Python 2.6 or 2.7 for clarity, since they're the only two versions.